lib/dal/src/workspace_snapshot/node_weight/func_node_weight.rs

```diff
@@ -100,7 +100,7 @@ impl FuncNodeWeight {
 
     pub fn node_hash(&self) -> ContentHash {
         ContentHash::from(&serde_json::json![{
-            "content_address": self.content_address,
+            "content_hash": self.content_address.content_hash(),
```

Comment:
These are _technically_ fine, since `serde_json` will basically output a form of the debug output of the enum, but it's such an easy thing to forget that it's special handling from `serde_json` that it felt like we should avoid using it directly for the `node_hash` in general.